package.json

```diff
@@ -157,18 +174,18 @@
     "version": "node ./scripts/version.js && git add CHANGELOG.md"
   },
   "devDependencies": {
-    "@types/mocha": "^2.2.43",
-    "@types/node": "^7.0.39",
-    "cross-env": "^5.0.5",
+    "@types/mocha": "^2.2.44",
+    "@types/node": "^7.0.48",
+    "cross-env": "^5.1.1",
     "mocha": "^3.5.3",
-    "typescript": "^2.4.2",
-    "vscode": "^1.1.6"
+    "typescript": "^2.6.2",
+    "vscode": "^1.1.10"
   },
   "dependencies": {
+    "ignore": "^3.3.7",
+    "prettier": "1.9.0",
+    "prettier-eslint": "^8.2.4",
```

Comment:
@RobinMalfait  8.2.0 -> 8.2.4
There is also a 8.2.5 now

Reply:
Whoops, didn't see that!